src/Adapter/Driver/Oci8/Oci8.php

```diff
@@ -34,23 +41,40 @@ class Oci8 implements DriverInterface, Profiler\ProfilerAwareInterface
      * @var Profiler\ProfilerInterface
      */
     protected $profiler = null;
+    /**
+     * @var array
+     */
+    protected $options = [];
+    /**
+     * @var array
+     */
+    protected $features = [];
 
     /**
      * @param array|Connection|\oci8 $connection
      * @param null|Statement $statementPrototype
      * @param null|Result $resultPrototype
+     * @param array $options
      */
-    public function __construct($connection, Statement $statementPrototype = null, Result $resultPrototype = null)
+    public function __construct($connection, Statement $statementPrototype = null, Result $resultPrototype = null, array $options = [], $features = self::FEATURES_DEFAULT)
     {
         if (!$connection instanceof Connection) {
             $connection = new Connection($connection);
         }
-
+        $options = array_intersect_key(array_merge($this->options, $options), $this->options);
         $this->registerConnection($connection);
         $this->registerStatementPrototype(($statementPrototype) ?: new Statement());
         $this->registerResultPrototype(($resultPrototype) ?: new Result());
+        if (is_array($features)) {
+            foreach ($features as $name => $feature) {
+                $this->addFeature($name, $feature);
+            }
+        } elseif ($features instanceof AbstractFeature) {
+            $this->addFeature($features->getName(), $features);
+        } elseif ($features === self::FEATURES_DEFAULT) {
+            $this->setupDefaultFeatures();
+        }
     }
-
```

Comment:
I'm pretty sure these are a result of running `php-cs-fixer` without specifying the configuration file. Unfortunately, `php-cs-fixer`, when run a second time *with* the configuration file, will not fix them. We can fix these on merge.